skynet/modules/ttt/openai_api/app.py

```diff
@@ -23,16 +26,28 @@ def initialize():
 
     global proc
 
-    proc = subprocess.Popen(
-        f'{openai_api_server_path} \
-            --batch-size {llama_n_batch} \
-            --ctx-size {llama_n_ctx} \
-            --flash-attn \
-            --model {llama_path} \
-            --n-gpu-layers {llama_n_gpu_layers} \
-            --port {openai_api_server_port}'.split(),
-        shell=False,
-    )
+    if use_vllm:
+        openai_api_server_path = vllm_server_path
+        proc = subprocess.Popen(
```

Comment:
@saghul this allows us to restart just the api server when something times out

Reply:
No strong opinion, we can always revisit after we gain more operational experience with vllm.